net/autossh/src/opnsense/mvc/app/controllers/VerbNetworks/Autossh/forms/tunnels.xml

```diff
@@ -0,0 +1,200 @@
+<form>
+    <field>
+        <id>tunnel.enabled</id>
+        <label>Enabled</label>
+        <type>checkbox</type>
+        <help>Enable this Autossh tunnel.</help>
+    </field>
+    <field>
+        <id>tunnel.description</id>
+        <label>Description</label>
+        <type>text</type>
+        <help><![CDATA[Free form text field available to describe this connection if required.]]></help>
```

Comment:
No cdata needed

Reply:
okay - CDATA removed from the fields where it is not needed